### PR TITLE
feat(data-warehouse): Add `created_at` to BigQuery persons batch export

### PR DESCRIPTION
**Pull request by rossgray**
## Problem

We don't currently send the `created_at` field in the persons batch export. Here we add it for the BigQuery destination (others to follow).

## Changes

- Updated the persons batch export view query (requires a ClickHouse migration)
- Updated the query used by BigQuery persons batch export to include the `created_at` column
- The `created_at` column will only be used when creating new tables, if the export uses a merge then it will be ignored in order to preserve the current table schema. If the user adds the column to their BigQuery table manually, the data will be included in subsequent batch exports.

## Questions

- Should I add the migration in a separate PR? I'm not sure if we could run into a case where the migration hasn't been applied but exports are running and failing due to the `created_at` column not existing yet?
- I wasn't able to test the frontend changes because I think persons batch export is behind a feature flag and not sure how to set that up locally. It's a fairly simple change though so I presume it should be ok?

## Post-deployment

- Need to update the batch export docs to mention the addition of this new field and also to describe how to pull in this field for existing batch exports

## Does this work well for both Cloud and self-hosted?

Should do

## How did you test this code?

Added unit test to cover both newly created tables and existing ones.

### CODE REVIEW

File: posthog/temporal/batch_exports/bigquery_batch_export.py [hunk 1]
```diff
@@ -174,48 +173,56 @@ async def adelete_table(
...
-        table_schema: list[bigquery.SchemaField],
+        table_schema: list[bigquery.SchemaField] | None = None,
         exists_ok: bool = True,
         not_found_ok: bool = True,
         delete: bool = True,
         create: bool = True,
     ) -> collections.abc.AsyncGenerator[bigquery.Table, None]:
         """Manage a table in BigQuery by ensure it exists while in context."""
         if create is True:
+            assert table_schema is not None, "Table schema is required when creating a table"
```
Comment (tomasfarias) [inline posthog/temporal/batch_exports/bigquery_batch_export.py:210]: Assertions are removed by optimized Python, I'm wondering if we would prefer to still keep the check by instead raising a `ValueError("Table schema is required when creating a table")` or other non-retryable error here.

Do we ever expect schema to be `None`?

**Review by tomasfarias — CHANGES_REQUESTED** [review]
The migration will need to be bumped to 0093 (or later) as we'll merge my #26477 first. But otherwise PR looks good!

File: posthog/temporal/batch_exports/bigquery_batch_export.py (hunk 1 above)
Comment (rossgray) [reply posthog/temporal/batch_exports/bigquery_batch_export.py:210]: > Assertions are removed by optimized Python
ah right, I wasn't aware of this, thanks!

> Do we ever expect schema to be `None`?
To be honest, we're always called `managed_table` with a schema so we can probably remove this check

**Review by tomasfarias — APPROVED** [review]

**Review by Daesgar — APPROVED** [review]
lgtm!

**Pull request merged**